2FirstWeeks/Grading/Program.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Grading
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+              List<int> GradingStudents(List<int> grade)
+            {
+                for (int i = 0; i < grade.Count; i++)
+                {
+                    if (grade[i] != 100 && grade[i] >= 38)
+                    {
+                        if (grade[i] == 38)
+                        {
+                            grade[i] += 2;
```

Comment:
```suggestion
                            grade[i] += 2;
```
k baoh được modify param truyền vào nhé. e tạo 1 List mới rồi return lại

Reply:
@TienDatNguyen321 chỗ này e chưa fix nhé